### ISSUE TITLE
stop moving me plz

### ISSUE THREAD

**Issue by Popoteguitou** [pokeclicker-dev/pokeclicker#115]
when you exit the farm/daycare/underground, the game put you on the route the location is... it's logic, but anoying

**Comment by Claygame** [comment]
haha, yeah this annoy the heck out of me too, makes me not go there most of the time. It is a godsend that you get the List button with area2 and you dont have to go to route5 alllll the time xD

**Comment by NikafieNd** [comment]
This only happens on the Underground and Farm.
Clicking the little Daycare square will not move you to Route 5.

**Comment by RedSparr0w** [comment]
Think the daycare only stops moving you once you get to the second region.

Maybe we could implement something similar for Underground/Farm when we add shortcuts to those.

**Comment by Claygame** [comment]
yeah sounds good. but these need a way to access in regions even if there is no shortcut unlocked yet.
